packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -1231,6 +1230,63 @@ export default abstract class SqlIntegration
   async getExperimentResults(): Promise<ExperimentQueryResponses> {
     throw new Error("Not implemented");
   }
+  getInformationSchemaFromClause(): string {
+    return "information_schema.columns";
+  }
+  getInformationSchemaWhereClause(): string {
+    return "table_schema NOT IN ('INFORMATION_SCHEMA')";
```

Comment:
Should this be uppercase like this?  I think this was lowercase in all the queries before.

Reply:
So, this was done for Snowflake, as they use standard, case-sensitive SQL comparison semantics - if this is lowercased, we get two schemas when testing, one if which is `INFORMATION_SCHEMA` since it's not getting filtered out. 

I've udpated this so the standard `getInformationSchemaWhereClause` returns `information_schema` and I'm now overriding that within `Snowflake.ts`. This pattern feels a bit better.